### ISSUE TITLE
Redirect to Home Page with miniplayer

### ISSUE THREAD

**Issue by kutibotond** [proffapt/own-youtube#36]
My request is to simply add an option to the "Redirect the Homepage" menu that would switch on the miniplayer while showing the homepage, just like pressing the "m" key would. That way, when you watch a video and click on the Youtube logo it doesn't interrupt the video and you can browse the homepage.

**Comment by proffapt** [comment]
Hi! 
First of all apologies for the delay - _a long one_.

I got involved into college and other activities that I lost track of it. I am planning to learn more and revamp the project ( Complete UI revamp ). I have set around 2-3 months of span for the learning (after completing current immediate things at hand). Will handle the Feature request during that time. Will announce it when I start revamping it, so that others interested can give me some _fingers_ .

The goal of having all in one extension for youtube is still strong as before, I just can't handle 4-5 extensions just to tweak youtube.
